applications/band-grants.md

```diff
@@ -0,0 +1,164 @@
+>
+> See the [Grant Program Process](https://github.com/smart-contract-bounty/Wasm-Bounty-01#pencil-process) on how to submit a proposal.
+- **Team Name:** World Data Corporation Foundation (“Band Protocol”)
+- **Payment Address:** 12aj1nedSqMsVhAH5LyYJtjKPNrFWYoEh4FeGmWW2aS4DQQh
+
+
+> :exclamation: *The combination of your GitHub account submitting the application and the payment address above will be your unique identifier during the program. Please keep them safe.*
+
+## Project Overview :page_facing_up:
+### Overview
+
+Please provide the following:
+
+- If the name of your project is not descriptive, a tag line (one sentence summary).
+    - Band Protocol is a leading cross-chain data oracle provider that aggregates and connects real-world data and APIs to smart contracts.
+- A brief description of the template building.
+    - BandChain is based on Cosmos SDK, designed to be compatible with all smart contract platforms and blockchain development frameworks. In a trustless and decentralized manner, BandChain does all the heavy lifting jobs of pulling data from external sources, aggregating, and packaging them into a format that is easy to use and cryptographically verified across multiple blockchains. Unlike general-purpose blockchains, BandChain is designed explicitly for oracle data requests and computation. Data request transactions are both received and resolved very quickly. Furthermore, our data source scripts and oracle scripts allow maximum customization and flexibility for the user to query and compute their desired data feed.
+- An indication of how your project relates to ink! ecosystem.
+    - Band Protocol has the potential to become a crucial infrastructure provider for the ink! Ecosystem. As projects develop on Astar, they will require reliable oracle services to deliver precise real-world data, and VRF to produce randomized results in their gaming and lottery mechanisms. By integrating with Astar, we can prepare and offer these essential services to Dapps building on the platform. Additionally, developers can be assured that they will receive high-quality data from a team of experienced professionals managing the price feed. By leveraging Band Protocol's expertise, Astar can enhance its offerings and provide a more comprehensive, reliable ecosystem for its users. Furthermore, we have a strong track record of collaborating with and offering oracle solutions for astar for more than a year now. This extensive experience positions us as a knowledgeable and skilled team in supporting the polkadot ecosystem and we are truly enthusiastic about continuing our involvement to its growth.
```

Comment:
Does Band plan to provide the full access to the standard dataset as well as the VRF features to ink contracts this time?

Reply:
We have the capability to offer up to 15 symbols from our standard dataset (https://data.bandprotocol.com/). The VRF will not be included in ink contracts at this time. Nevertheless, we are eager and willing to integrate our VRF into ink contracts in the near future.